recipes/qt/5.x.x/conanfile.py

```diff
@@ -662,6 +666,20 @@ def package(self):
         for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
             os.remove(fl)
 
+        for m in os.listdir(os.path.join(self.package_folder, "lib", "cmake")):
+            module = os.path.join(self.package_folder, "lib", "cmake", m, "%sMacros.cmake" % m)
+            if not os.path.isfile(module):
+                tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", m))
+                
+        cmake_executables_variables = textwrap.dedent("""\
+            set(Qt5Core_QMAKE_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/qmake)
+            set(Qt5Core_MOC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/moc)
+            set(Qt5Core_RCC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/rcc)""")
+        if self.options.widgets:
+            cmake_executables_variables.append("\nset(Qt5Widgets_UIC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/uic)")
```

Comment:
`Qt5::qmake`, `Qt5::moc`, `Qt5::uic` and `Qt5::rcc` executables imported targets could be easily added in this file along these variables:

```cmake
get_filename_component(Qt5Core_QMAKE_EXECUTABLE "${CMAKE_CURRENT_LIST_DIR}/../../../bin/qmake" ABSOLUTE)
if(NOT TARGET Qt5::qmake)
    add_executable(Qt5::qmake IMPORTED)
    set_property(TARGET Qt5::qmake PROPERTY IMPORTED_LOCATION ${Qt5Core_QMAKE_EXECUTABLE})
endif()
...
```

Reply:
My plan was to add these in a similar fashion as https://github.com/conan-io/conan-center-index/pull/4689, once it is merged, to avoid double-review.

Reply:
> I'm also wondering if adding `.exe` extension on Windows is not more robust.

I added such imported targets by myself and it doesn't work without `.exe` extension on Windows.